### DISCUSSION TITLE
July 29 Quansight Show & Tell

### DISCUSSION THREAD

**Discussion by isabela-pf** [Quansight/Quansight.github.io#36] [Show and tell]
July 29 show & tell notes. Please use this discussion to mark highlights and links from our discussions.

DO NOT DISCLOSE CONFIDENTIAL CUSTOMER INFORMATION

**Comment by tonyfast** [comment]
* meson builds - https://labs.quansight.org/blog/2021/07/moving-scipy-to-meson/
* wason selection task  - https://en.wikipedia.org/wiki/Wason_selection_task
* rise - https://rise.readthedocs.io/en/stable/
* jax - https://jax.readthedocs.io/en/latest/jax.numpy.html
* jax colab - https://colab.research.google.com/drive/1CRVTRG8g6s6GL1L8g6iOkfg8MbzTKsc2#scrollTo=QPBZUlApSiy9
* zarr - https://zarr.readthedocs.io/en/stable/
* mimesis random data generator - https://mimesis.readthedocs.io/
* hypothesis examples - https://hypothesis.readthedocs.io/en/latest/data.html

Colab hover state for rtc ![image](https://user-images.githubusercontent.com/4236275/127532290-1e8dfa60-0cc2-494f-844f-2d5e5feebf15.png)

* binder with rtc and jupyter video chat https://github.com/deathbeds/hourhaus

**Reply by dhavide** [reply]
Wason's Selection (4-Card) task:

+ https://en.wikipedia.org/wiki/Wason_selection_task
+ https://mathlair.allfunandgames.ca/fourcard.php
+ https://www.socialpsychology.org/teach/wason.htm

Arguments about avoiding Foo bar:
+ https://www.crondose.com/2018/04/foo-bar-definition-why-you-should-not-use-it/
+ https://www.techyourchance.com/never-use-foo-bar-baz/
